decorate-angular-cli.js

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
Is this file autogenerated ?

Reply:
Yep, this was added by the NX CLI migrations.